src/FormControls/RadioButton.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default class RadioButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getDOMNode() {
+    return React.findDOMNode(this.refs.input);
+  }
+
+  getValue() {
+    return this.props.value;
+  }
+
+  getChecked() {
+    return this.getDOMNode().checked;
+  }
+
+  renderLabel(input) {
+    if (!this.props.label) {
+      return input;
+    }
+
+    const classes = {
+      'radio': !this.props.inline,
+      'radio-inline': this.props.inline
+    };
+
+    return (
+      <label className={classNames(classes)}>
+        {input}
```

Comment:
the label should probably also add `htmlFor` if there is an id prop, for better accessibility. We might also want to add `id` as a `requiredForA11y` prop to encourage good behavior. I wouldn't assume the prop is there tho to avoid breaking compatibility